src/Compilers/CSharp/Portable/Syntax/SyntaxNodeRemover.cs

```diff
@@ -260,6 +288,19 @@ public override SeparatedSyntaxList<TNode> VisitList<TNode>(SeparatedSyntaxList<
                 return list;
             }
 
+            private bool ContainsEndOfLine(SyntaxTriviaList triviaList)
```

Comment:
Method can be `static` or perhaps an instance method on `SyntaxTriviaList`. Or alternatively, replace with `triviaList.IndexOf(SyntaxKind.EndOfLineTrivia) >= 0`.